modules/generative-openai/module.go

```diff
@@ -66,8 +66,9 @@ func (m *GenerativeOpenAIModule) initAdditional(ctx context.Context,
 ) error {
 	openAIApiKey := os.Getenv("OPENAI_APIKEY")
 	azureApiKey := os.Getenv("AZURE_APIKEY")
+	openAIBaseUrl := os.Getenv("OPENAI_BASE_URL")
```

Comment:
We should expose changing of BaseURL only through class settings. Just should be removed then